global_planner_node:

  show_config: true
  debug: true
  
  # ws_x_max: 43.5
  # ws_y_max: 42.5
  # map_resolution: 0.05
  traj_dxy_max: 1
  traj_pos_tol: 10
  traj_yaw_tol: 0.1
  goal_weight: 1.5
  cost_weight: 0.25
  lof_distance: 1
  occ_threshold: 100

  world_frame: map
  robot_base_frame: base_link

  traj_topic: /trajectory_tracker/input_trajectory
  vis_marker_traj_topic: /global_planner_node/visualization_marker_trajectory
  goal_topic: /move_base_simple/goal
  #global_map_topic: /global_planner_node/global_costmap/costmap #/costmap_2d/costmap/costmap
